content/manuals/engine/network/drivers/host.md

```diff
@@ -93,19 +104,66 @@ $ nc localhost 80
 ### Limitations
 
 - Processes inside the container cannot bind to the IP addresses of the host
- because the container has no direct access to the interfaces of the host.
+  because the container has no direct access to the interfaces of the host.
 - The host network feature of Docker Desktop works on layer 4. This means that
-unlike with Docker on Linux, network protocols that operate below TCP or UDP are
-not supported.
+  unlike with Docker on Linux, network protocols that operate below TCP or UDP are
+  not supported.
 - This feature doesn't work with Enhanced Container Isolation enabled, since
-isolating your containers from the host and allowing them access to the host
-network contradict each other.
+  isolating your containers from the host and allowing them access to the host
+  network contradict each other.
 - Only Linux containers are supported. Host networking does not work with
   Windows containers.
 
+## Usage example
+
+This example shows how to start an nginx container that binds directly to port
+80 on the Docker host. From a networking perspective, this provides the same
```

Comment:
```suggestion
`80` on the Docker host. From a networking perspective, this provides the same
```

Reply:
I'm not sure about wrapping port numbers in code spans. They are not really code (I guess IP addresses are not really code either) but I feel like port numbers are less annoying to read if they are formatted as normal text. They are just integers between 0 and 65535 :-)